pkg/operator/ceph/cluster/nodedaemon/exporter.go

```diff
@@ -120,6 +120,7 @@ func (r *ReconcileNode) createOrUpdateCephExporter(node corev1.Node, tolerations
 			controller.AddCephVersionLabelToDeployment(*cephVersion, deploy)
 		}
 		var terminationGracePeriodSeconds int64 = 2
+		deploy.Spec.Strategy.Type = appsv1.RecreateDeploymentStrategyType
```

Comment:
add a comment why we are adding rolling update, explaining about socket file

Reply:
ack